src/TAction.h

```diff
@@ -94,6 +94,9 @@ class TAction : public Tree<TAction>, public QObject
     void setDataSaved() { if (mpParent) { mpParent->setDataSaved(); } mDataChanged = false; }
     void setDataChanged() { if (mpParent) { mpParent->setDataChanged(); } mDataChanged = true; }
     bool isDataChanged() { return mDataChanged; }
+    QString packageName(TAction* pAction);
+    QString moduleName(TAction* pAction);
```

Comment:
These don't modify the class members do they - so they could be declare thusly:
```suggestion
    QString packageName(TAction* pAction) const;
    QString moduleName(TAction* pAction) const;
```